Makefile

```diff
@@ -62,3 +71,14 @@ build: clean
 .PHONY: clean
 clean:
 	rm -rf dist
+
+
+# full hard reset
+.PHONY: nuke
+nuke:: local-uninstall clean
```

Comment:
I do not really like the name `nuke`. And why does it have two colons instead of one?

Reply:
Rationale:
I wanted: "make it new" but can't use spaces
Why 'nuke'; english word-play: "The name "NUKE" originates from a combination of the words _New_ and _Make_." (eg. NU + KE)

However, if you insist on another name:
alternatives are the longer: [GNU's maintainers-clean](https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html) or perhaps "it-new" ?

---

Rationale:
[double colons in make](https://stackoverflow.com/q/7891097) are ideal for asynchronous resets like this.

Reply:
Just learned that apparently the double colons execute even if none of the requirements changes.

I am not sure about the best name, but for me even something simple like `full-clean` would be sufficient.

Reply:
I concur.

```suggestion
full-clean:: local-uninstall clean
```